misc/net.minetest.minetest.appdata.xml

```diff
@@ -16,31 +16,31 @@
 		<p>
 			Minetest is an infinite-world block sandbox game and game engine.
 		</p><p xml:lang="de">
-			Minetest ist ein Sandkastenspiel und Spielengine mit unendlichen Welten
+			Minetest ist ein Sandkastenspiel und eine Spielengine mit unendlichen Welten.
```

Comment:
Is there really no German word to express game engine? I'm not 100%, but wouldn't it be "Spiel-Engine" if Deutschified?

Reply:
wp has the spelling with a hyphen https://de.wikipedia.org/wiki/Spiel-Engine